wordpress-mobile/WordPress-iOS
# Offline Mode: Remove secondary actions from the editor

- Update the primary action for drafts to always be “Publish” (to match web and Android)
- Remove secondary actions (“Save as Draft”) – you tap back now and it’s offline-first

This eliminates the scenario where you tap "Update" and need to force the sync engine to save the changes synchronously, so we no longer have to worry about it.

## To test:

- **Verify** that both new and existing drafts have a primary action "Publish" and no secondary action

The secondary action used to be here:

<img width="320" alt="Screenshot 2024-03-27 at 9 14 24 AM" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1567433/18a8f6e4-658a-4ca0-9d7c-37197de4a8eb">


## Regression Notes
1. Potential unintended areas of impact: Post Editor
2. What I did to test those areas of impact (or what existing automated tests I relied on): n/a
3. What automated tests I added (or what prevented me from doing so): n/a

PR submission checklist:


Testing checklist:
